runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkExecutionEnvironments.java

```diff
@@ -67,6 +69,7 @@ static ExecutionEnvironment createBatchExecutionEnvironment(
 
     // depending on the master, create the right environment.
     if ("[local]".equals(flinkMasterHostPort)) {
+      flinkConfiguration.set(TaskManagerOptions.MANAGED_MEMORY_SIZE, MemorySize.parse("2048m"));
```

Comment:
It has sets the default managed memory size to 128MB for MiniCluster in https://issues.apache.org/jira/browse/FLINK-15763. Have set it to a large value when the master host is [local]. Appreciate for any suggestions on a better way to address this issue.